### DISCUSSION TITLE
【模拟电路】【物料推荐】在JFET逐渐消逝的年代选择一只（一对）合适的JFET

### DISCUSSION THREAD

**Discussion by alwaysmy** [emoestudio/eEEExplore-2023#11] [General]
~~跑题：这个discussion的功能似乎专门为论坛而生~~

好了不跑题。这个标题其实不是我的推荐，而是我的疑惑，当然在这个讨论最后我列出我看过的一些总结。
这个话题的直接起因是，有网友发出一篇BF862开盖的链接，表示tb上的器件可信度未知，而在这之前我恰好推荐了两毛八的BF862----我开始考虑，这个东西是不是假货？

那么这个discussion的主题是:

1. 还有什么型号的JFET依然大量在产且能买到真货？
2. 如果没有稳定渠道，如何较为容易地鉴定？
   

> 低情商：有什么一块钱一下的便宜的管子给我推荐推荐。

其实更详细来说就是：

第一，设计中能不能用较为便宜的JFET替代更贵的管子，例如叠加是可以考虑的。
以2sk208 2sk209为例，在lcsc上按照销量排列jfet，除去开关管，剩下就可以看到拍在最前面的就剩这几个（按照销量来排是因为考虑到用的人多，价格会比较便宜）。在lcsc上批量价格6毛到9毛，少量也是一元左右。

但是，根据某些实测测试结果表示，似乎存在效果更好而且更便宜的管子，这时候我的手欠就开始了，问题也随之而来，在选型初期如何快速确定买到的管子的效果呢？例如如何快速测定参数。

> 这里不说是否假货因为我考虑到抓到老鼠都是好猫

当然最直接的办法就是直接塞到设计上对比是否满足设计要求和仿真设计差距，但是还有些问题就在于，如果手上没有确定的设计，如果有问题，如何能确定是否是自己的问题（

当然，还有一个就是由网友发的，直接开盖的方法：这里贴出这个链接[BF862真假对比](https://zeptobars.com/en/read/BF862-low-noise-N-Channel-JFET-J-FET-AM-preamp-2Ap)

这个链接中，验证了假货的丝印是2Ap, 在淘宝上确实存在少量卖这个丝印的店家声称这是BF862, 而真货就是2AM, 这个丝印在淘宝上普遍比较贵。

其实这个讨论放在这里也就是说，如果有现成的结果，能发在这里分享一下是再好不过啦！

> 这就开始白嫖

俺也不能光做伸手党，下面详细说一下我的经历和推荐。
我从关注这个主要是因为捡垃圾的时候顺手带了几个电容式麦克风，不是那种驻极体的，就是单纯的电容式麦克风，然后我就寻思hifi一把，做到一半发现这玩意儿输出信号非常小，一不小心信噪比就掉了。买来的垃圾上自带的pcb是用的2sk596.

这个是上世纪基本上可以专为大众消费级音频话筒方案而用的一个JFET, 价格低廉，仿制一堆，现在在TB上还是几毛钱一个，然而，它的性能本身就是一般，更何况国产各种各样。

-----废话说太多了，这里直接快进到我买了BF862之后发现和正版丝印不一样然后尝试开盖的经历，我的是丝印W28U：

![1689943815961.png](https://img1.imgtp.com/2023/07/21/BRBWMwaf.png)

![1689943894109.png](https://img1.imgtp.com/2023/07/21/mQTVrKAA.png)

首先，我是完全不懂开盖（

希望请教了解的人，开盖之后还需要什么操作什么吗，因为我觉得这上面似乎看起来不应该有这么多似乎是用于绑定线的点？

在上面那个BF862开盖链接里面两个型号都是只有两根引线，这里似乎对不上了。

其次，die的大小就是这一片的大小吗，如果是的，那么正版BF862的Die是一个正方形表面，而这显然是一个长方形表面，如果是正品不同的批次会有这个区别吗

最后，BF862在手册上指出DS 是完全对称的可互换，但是看这个图似乎并不是对称的设计。

如果我做的完全不对，希望大佬来指教~

> 更希望有大佬直接甩出他的大设备进行激情地拍摄，甩出高清大图

# 最后给出一些总结

不少是来自于网友测试和使用，俺大言不惭的先列出来了，如果网友们来讨论我会@出来。

大致从我觉得靠谱到不靠谱或者太贵排列，主要依据是低噪声，如果大伙有更多的其他需求，俺也想看看你们的对比。

- 2sk208 209，lcsc有售，一元左右，某宝也有售但是似乎更贵。。其中208 NF为0.5db

- BF861 LCSC标记20.。。某宝只有两家 价格在1.2-3.5rmb. 没有试过，下面是它的等效输入噪声。

![1689942123119.png](https://img1.imgtp.com/2023/07/21/hDObILlj.png)

当我正准备说大受震撼的时候再次发现单位是kHz. 但是依然还凑合？问题是在digikey上标记这个是停产？？？

NXP上说这个还在产啊？同时这个器件没有官方提供的仿真模型。

- BF862 前面已经说过了，明确停产，所以不是很推荐了倒是可以买一些试一试，因为确实便宜

- LSK170 LSJ74B 互补JFET，网友鼎立推荐，但是我找不到合理的链接了，某宝上的那些，我觉得依然需要验证

- 2SK170 2SK117, 我买过2SK117,实测中我觉得是假的，但是这种hifi重灾区我觉得不太可能是真货。。。排出来这俩是因为能买到而且有测试噪声比较低，最大问题就是不保真。。。在仿真设计中我觉得2sk117的性能非常逆天。
  
  

- JFE150 某宝12以上，也可以从TI官方买
  其中等效输入噪声：
  ![1689941750843.png](https://img1.imgtp.com/2023/07/21/CuZjFDyN.png)

- JFE2140 ,替代LSK389,某宝25以上,也可以从TI官方买

- 2sk211 假货泛滥
## 性能不确定

>  或者说确定不太行但是在产能买到。。。

所以这部分用来做个记录：

- 我从NXP网站上筛选了一下，看到了一个BF510-513系列是标记为低噪声的，然而NF 1.5db，$g_m$也并不大。主要是明确有货但是其实也不好买。

NXP的剩下的在产的抛开开关管，其他的没一个能打的了，这里就不一一记录了。

- TF2123, 看结构似乎就是用来替代2SK596的，性能还不如正版的2SK596,估计用于低端音频应用较多，所以便宜，LCSC上才2毛一个。

- 2sk508g lcsc价格：6毛，gm 最小14，噪声没标出来。

- 2sk596 确定是用于低端应用，但是便宜，批量在产，没事儿的时候可以考虑用用。

# 后续

考虑学一下这篇博客做个夹具测一下：[对比测试噪声](http://tech-blog-en.sblo.jp/article/183391570.html)
但是还是要先把NAP做出来。。。我实在是太懒惰了。。。。

**Comment by OliverKung** [comment]
关于片子验真假的问题，感觉开盖是一个相当靠谱的方案。管子的话可能不太好区分，片子的管芯上一般会有明显的型号代码和厂商编号。但是怎么方便简单的开盖又是一个问题了。硝酸方法要三算很不好买。机械打磨很容易就过了。激光好像是个不错的方案，但是啥样的激光器比较适合开盖的操作呢？

从 Windows 版邮件发送

发件人: alwaysmy
发送时间: 2023年7月21日 21:09
收件人: emoestudio/eEEExplore-2023
抄送: Subscribed
主题: [emoestudio/eEEExplore-2023]=?UTF-8?Q?_=E3=80=90=E6=A8=A1=E6=8B=9F=E7=94=B5=E8=B7=AF=E3=80=91=E3=80=90=E7=89=A9=E6=96=99=E6=8E=A8=E8=8D=90=E3=80=91=E5=9C=A8JFET=E9=80=90=E6=B8=90=E6=B6=88=E9=80=9D=E7=9A=84=E5=B9=B4=E4=BB=A3=E9=80=89=E6=8B=A9=E4=B8=80=E5=8F=AA=EF=BC=88=E4=B8=80=E5=AF=B9=EF=BC=89=E5=90=88=E9=80=82=E7=9A=84JFET?==?UTF-8?Q?_=28Discussion_#11=29?=

跑题：这个discussion的功能似乎专门为论坛而生
好了不跑题。这个标题其实不是我的推荐，而是我的疑惑，当然在这个讨论最后我列出我看过的一些总结。
这个话题的直接起因是，有网友发出一篇BF862开盖的链接，表示tb上的器件可信度未知，而在这之前我恰好推荐了两毛八的BF862----我开始考虑，这个东西是不是假货？
那么这个discussion的主题是:
1. 还有什么型号的JFET依然大量在产且能买到真货？
2. 如果没有稳定渠道，如何较为容易地鉴定？
低情商：有什么一块钱一下的便宜的管子给我推荐推荐。
其实更详细来说就是：
第一，设计中能不能用较为便宜的JFET替代更贵的管子，例如叠加是可以考虑的。
以2sk208 2sk209为例，在lcsc上按照销量排列jfet，除去开关管，剩下就可以看到拍在最前面的就剩这几个（按照销量来排是因为考虑到用的人多，价格会比较便宜）。在lcsc上批量价格6毛到9毛，少量也是一元左右。
但是，根据某些实测测试结果表示，似乎存在效果更好而且更便宜的管子，这时候我的手欠就开始了，问题也随之而来，在选型初期如何快速确定买到的管子的效果呢？例如如何快速测定参数。
这里不说是否假货因为我考虑到抓到老鼠都是好猫
当然最直接的办法就是直接塞到设计上对比是否满足设计要求和仿真设计差距，但是还有些问题就在于，如果手上没有确定的设计，如果有问题，如何能确定是否是自己的问题（
当然，还有一个就是由网友发的，直接开盖的方法：这里贴出这个链接BF862真假对比
这个链接中，验证了假货的丝印是2Ap, 在淘宝上确实存在少量卖这个丝印的店家声称这是BF862, 而真货就是2AM, 这个丝印在淘宝上普遍比较贵。
其实这个讨论放在这里也就是说，如果有现成的结果，能发在这里分享一下是再好不过啦！
这就开始白嫖
俺也不能光做伸手党，下面详细说一下我的经历和推荐。
我从关注这个主要是因为捡垃圾的时候顺手带了几个电容式麦克风，不是那种驻极体的，就是单纯的电容式麦克风，然后我就寻思hifi一把，做到一半发现这玩意儿输出信号非常小，一不小心信噪比就掉了。买来的垃圾上自带的pcb是用的2sk596.
这个是上世纪基本上可以专为大众消费级音频话筒方案而用的一个JFET, 价格低廉，仿制一堆，现在在TB上还是几毛钱一个，然而，它的性能本身就是一般，更何况国产各种各样。
-----废话说太多了，这里直接快进到我买了BF862之后发现和正版丝印不一样然后尝试开盖的经历，我的是丝印W28U：


首先，我是完全不懂开盖（
希望请教了解的人，开盖之后还需要什么操作什么吗，因为我觉得这上面似乎看起来不应该有这么多似乎是用于绑定线的点？
在上面那个BF862开盖链接里面两个型号都是只有两根引线，这里似乎对不上了。
其次，die的大小就是这一片的大小吗，如果是的，那么正版BF862的Die是一个正方形表面，而这显然是一个长方形表面，如果是正品不同的批次会有这个区别吗
最后，BF862在手册上指出DS 是完全对称的可互换，但是看这个图似乎并不是对称的设计。
如果我做的完全不对，希望大佬来指教~
更希望有大佬直接甩出他的大设备进行激情地拍摄，甩出高清大图
最后给出一些总结
不少是来自于网友测试和使用，俺大言不惭的先列出来了，如果网友们来讨论我会@出来。
大致从我觉得靠谱到不靠谱或者太贵排列，主要依据是低噪声，如果大伙有更多的其他需求，俺也想看看你们的对比。
• 2sk208 209，lcsc有售，一元左右，某宝也有售但是似乎更贵。。其中208 NF为0.5db
• BF861 LCSC标记20.。。某宝只有两家 价格在1.2-3.5rmb. 没有试过，下面是它的等效输入噪声。

当我正准备说大受震撼的时候再次发现单位是kHz. 但是依然还凑合？问题是在digikey上标记这个是停产？？？
NXP上说这个还在产啊？同时这个器件没有官方提供的仿真模型。
• BF862 前面已经说过了，明确停产，所以不是很推荐了倒是可以买一些试一试，因为确实便宜
• LSK170 LSJ74B 互补JFET，网友鼎立推荐，但是我找不到合理的链接了，某宝上的那些，我觉得依然需要验证
• 2SK170 2SK117, 我买过2SK117,实测中我觉得是假的，但是这种hifi重灾区我觉得不太可能是真货。。。排出来这俩是因为能买到而且有测试噪声比较低，最大问题就是不保真。。。在仿真设计中我觉得2sk117的性能非常逆天。
• JFE150 某宝12以上，也可以从TI官方买
其中等效输入噪声：

• JFE2140 ,替代LSK389,某宝25以上,也可以从TI官方买
• 2sk211 假货泛滥
性能不确定
或者说确定不太行但是在产能买到。。。
所以这部分用来做个记录：
• 我从NXP网站上筛选了一下，看到了一个BF510-513系列是标记为低噪声的，然而NF 1.5db，$g_m$也并不大。主要是明确有货但是其实也不好买。
NXP的剩下的在产的抛开开关管，其他的没一个能打的了，这里就不一一记录了。
• TF2123, 看结构似乎就是用来替代2SK596的，性能还不如正版的2SK596,估计用于低端音频应用较多，所以便宜，LCSC上才2毛一个。
• 2sk508g lcsc价格：6毛，gm 最小14，噪声没标出来。
• 2sk596 确定是用于低端应用，但是便宜，批量在产，没事儿的时候可以考虑用用。
后续
考虑学一下这篇博客做个夹具测一下：对比测试噪声
但是还是要先把NAP做出来。。。我实在是太懒惰了。。。。
